### ISSUE TITLE
Python3 NBI client: empty error messages

### ISSUE THREAD

**Issue by TheNetworkIsDown** [extremenetworks/ExtremeScripting#155]
I found that calling `addMacAddress()` and printing `session.message` in case of error returns an empty string.

You set

https://github.com/extremenetworks/ExtremeScripting/blob/5f3a8e6d139517e1136efad5aaf994d64d6cb81a/XMC_XIQ-SE/nbi_clients/Python3/XMC_NBI.py#L455

In the examples at the bottom however you use `session.message`

https://github.com/extremenetworks/ExtremeScripting/blob/5f3a8e6d139517e1136efad5aaf994d64d6cb81a/XMC_XIQ-SE/nbi_clients/Python3/XMC_NBI.py#L838

However, `session` represents the XMC_NBI object at that point. 

You either have to write `print(session.session.message)`

Or put the error message into `self.message` instead of `self.session.message`.
```
self.message = self.data['accessControl']['addMACToEndSystemGroup']['message']
```

Since you're setting up

https://github.com/extremenetworks/ExtremeScripting/blob/5f3a8e6d139517e1136efad5aaf994d64d6cb81a/XMC_XIQ-SE/nbi_clients/Python3/XMC_NBI.py#L61

in `__init__` I guess the second solution would be the correct one.

This applies to all the functions, not only `addMacAddress()`.

**Comment by mnikulski** [comment]
Hi,

indeed is here a issue I will fix ASAP.
thanks for using the class as well reporting the issue.

**Comment by ZdenekPala** [comment]
should be fixed by the commit #158
